datafusion-examples/examples/external_dependency/query-aws-s3.rs

```diff
@@ -64,7 +64,7 @@ async fn main() -> Result<()> {
     df.show().await?;
 
     // dynamic query by the file path
-    ctx.enable_url_table();
+    let ctx = ctx.enable_url_table();
```

Comment:
this example actually has the exact same bug I hit in https://github.com/apache/datafusion/issues/12551 -- namely the *returned* ctx has the table enabled, not the current one.

Reply:
Thanks for reminding this bug!